src/calculateRentalCost.js

```diff
@@ -26,9 +26,22 @@
  *
  * @return {number}
  */
-
 function calculateRentalCost(days) {
   // write code here
+  const price = 40;
+  const totalAmount = days * price;
+  const threeRentalDays = 3;
+  const sevenRentalDays = 7;
+  const threeDaysDiscount = 20;
+  const sevenDaysDiscount = 50;
```

Comment:
```suggestion
  const largeDiscountDays = 3;
  const smallDiscountDays = 7;
  const smallDiscount = 20;
  const largeDiscount = 50;
```
Немає сенсу в назви змінних вносити цифрові значення. Змінні для того і створюються, щоб потім можна було легко міняти значення. 
Якщо ти заміниш число 3 на 4, наприклад, то назва threeRentalDays, яка буде містити в собі число 4 - це буде не дуже логічно